### ISSUE TITLE
Desktop app version

### ISSUE THREAD

**Issue by undergroundwires** [undergroundwires/privacy.sexy#20]
- [x] Use [vue-cli-plugin-electron-builder](https://github.com/nklayman/vue-cli-plugin-electron-builder)
- [x] Automate releasing Windows executable on each GitHub release, see [uploading artifact to existing release](https://github.com/Roang-zero1/github-upload-release-artifacts-action)
- [x] Add link to desktop versions on webpage (can check if running inside electron electron/electron#2288)
- [x] Update privacy policy
- [x] Add icon in desktop version
- [x] On desktop version open all URLs in users default browser
- [x] Add [auto updates](https://www.electronjs.org/docs/api/auto-updater)

**Comment by Disk2019** [comment]
Ausm Brother .
Eagerly waiting .
I have done it with f2ko bat to exe with a custom privacy icon & working great . tested that on a clean installed vm guest os win10 2004 build 19041.331.

https://www.f2ko.de/programme/bat-to-exe-converter/

**Comment by undergroundwires** [comment]
I understand what you want now. I was thinking about leveraging "privacy.sexy" as an executable, not the scripts. But it's probably step 1 because application requires some compute power to be able to generate executables and now it runs totally serverless

 So what you want can be a desktop-version only feature because desktop version will have access to your windows computer to be able to utilize e.g. `iexpress.exe` to generate executables (see [the solution here](https://superuser.com/questions/868340/how-can-i-convert-a-windows-batch-script-to-a-exe)). It can run then `iexpress.exe` to convert `bat` into `exe`. 

So when this issue is done, it can be interesting to have one more export option (to exe) in the desktop version of `privacy.sexy`. But it will be a separate issue after this one.

**Comment by undergroundwires** [comment]
I understand what you want now. I was thinking about leveraging "privacy.sexy" as an executable, not the scripts. But it's probably step 1 because application requires some compute power to be able to generate executables and now it runs totally serverless and I couldn't find any browser only way to convert `bat´ to `exe` without calling any windows APIs.

 So what you want can be a desktop-version only feature because desktop version will have access to your windows computer to be able to utilize e.g. `iexpress.exe` to generate executables (see [the solution here](https://superuser.com/questions/868340/how-can-i-convert-a-windows-batch-script-to-a-exe)). It can run then `iexpress.exe` to convert `bat` into `exe`. 

So when this issue is done, it can be interesting to have one more export option (to exe) in the desktop version of `privacy.sexy`. But it will be a separate issue after this one.

**Comment by undergroundwires** [comment]
I understand what you want now. I was thinking about leveraging "privacy.sexy" as an executable, not the scripts. But it's probably step 1 because application requires some kind of server to be able to generate executables and now it runs totally serverless and I couldn't find any browser only way to convert `bat´ to `exe` without calling any windows APIs.

So what you want can be a desktop-version only feature because desktop version will have access to your windows computer to be able to utilize e.g. `iexpress.exe` to generate executables (see [the solution here](https://superuser.com/questions/868340/how-can-i-convert-a-windows-batch-script-to-a-exe)). It can run then `iexpress.exe` to convert `bat` into `exe`. 

So when this issue is done, it can be interesting to have one more export option (to exe) in the desktop version of `privacy.sexy`. But it will be a separate issue after this one.

**Comment by undergroundwires** [comment]
[auto-updater](https://www.electronjs.org/docs/api/auto-updater) creates a release with the artifacts. [bump-everywhere](https://github.com/undergroundwires/bump-everywhere) also creates a release. We should create a release once maybe from `autoUpdates` and set [bump-everywhere](https://github.com/undergroundwires/bump-everywhere) to not create a release. Blocked by: undergroundwires/bump-everywhere#5

**Comment by undergroundwires** [comment]
Could not `autoUpdater` to show notifications on Windows 10. So it'll just update the program to the latest after quitting.

Tried following:
- set appId: `setAppUserModelId` or as [build configuration](https://www.electron.build/configuration/configuration) in `vue.config.js`
- set default protocol client: `setAsDefaultProtocolClient`
- import `Notification` through `import { Notification } from "electron"`

Related to: electron-userland/electron-builder#2700, electron/electron#10864 (still open status)

**Comment by undergroundwires** [comment]
Desktop versions for Windows + Linux + macOS are ready with autoupdates and will be release from now on 🥳. The first release with desktop support is [0.6.0](https://github.com/undergroundwires/privacy.sexy/releases/tag/0.6.0)  🎉
